### DISCUSSION TITLE
Removal of abandoned carts

### DISCUSSION THREAD

**Discussion by Linny3D** [vendurehq/vendure#1495] [Q&A] [answered]
Hi,
is there a way to completely remove carts after, let's say, a given amout of time?
I know you can manually cancel an active Order, but that is not the same. They are still cluttering the "completed" list in the Admin ui, marked as "Cancelled".
After some time there will be a huge amount of them, as there will always be customers, which add products to the cart, but will never purchase them.

I didn't find anything about this is the docs. Maybe i'm blind though ;)

Cheers
Linny

**Comment by michaelbromley** [comment, marked as answer]
Hi,

This is not handled by default, but there is this issue: https://github.com/vendure-ecommerce/vendure/issues/1425 which aims to design a mechanism to handle this (and other similar scenarios).

**Reply by Linny3D** [reply]
Thank you!
